.markdownlint-cli2.cjs

```diff
@@ -0,0 +1,30 @@
+const githubMarkdownOpinions = require('@github/markdownlint-github')
+
+// Rules we want to turn on but currently have too many violations
+const rulesToEnforce = {
+  'fenced-code-language': false,
+  'no-duplicate-header': false, // Fix https://github.com/primer/doctocat/issues/527, then set this rule to `siblings_only: true`
+}
+// Rules we don't care to enforce (usually stylistic)
+const rulesToNotEnforce = {
+  'line-length': false,
+  'blanks-around-headings': false,
+  'blanks-around-lists': false,
+  'no-trailing-spaces': false,
+  'no-multiple-blanks': false,
+  'no-trailing-punctuation': false,
+  'single-trailing-newline': false,
+  'ul-indent': false,
+  'no-hard-tabs': false,
+  'first-line-heading': false,
+  'no-space-in-emphasis': false,
+  'blanks-around-fences': false,
+}
```

Comment:
These are stylistic rules that I don't think Primer team cares to enforce for now, but not sure. Feel free to update this.